kolibri/content/api.py

```diff
@@ -0,0 +1,222 @@
+"""
+This module acts as the only interface point between other apps and the database backend for the content.
+It exposes several convenience functions for accessing content
+"""
+from kolibri.content import models as KolibriContent
+from uuid import UUID
+import os
+from bulk_update.helper import bulk_update
+from django.core.files import File as DjFile
+
+"""helper funcitons"""
+
+def is_valid_uuid(uuid_to_test, version=4):
+    """
+    Check if uuid_to_test is a valid UUID.
+
+    Parameters
+    ----------
+    uuid_to_test : str
+    version : {1, 2, 3, 4}
+
+    Returns
+    -------
+    `True` if uuid_to_test is a valid UUID, otherwise `False`.
+
+    Examples
+    --------
+    >>> is_valid_uuid('c9bf9e57-1685-4c89-bafb-ff5af830be8a')
+    True
+    >>> is_valid_uuid('c9bf9e58')
+    False
+    """
+    try:
+        uuid_obj = UUID(uuid_to_test, version=version)
+    except:
```

Comment:
There is no exception to the rule that you should never catch all exceptions :) Only capture named exceptions with an explicit purpose.